_modules/ext_module_qvm.py

```diff
@@ -891,12 +867,29 @@ def run_post(cmd, status, data):
         # Execute command (will not execute in test mode)
         data = dict(key=dest, value_old=value_current, value_new=value_new)
         # pylint: disable=W0212
-        cmd = '/usr/bin/qvm-prefs {0} --set {1} {2} "{3}"'.format(
-            ' '.join(args._arg_info['_argparse_flags']), args.vmname, dest,
-            value_new
-        )
+        if value_new is not None:
+            print "Setting %s to %s" % (dest, value_new)
+            if dest == 'pcidevs':
+                for dev_id in value_new:
+                    dev = qubes.devices.PCIDevice(dev_id.strip())   
```

Comment:
I'll probably break this one as part of QubesOS/qubes-issues#2257, but don't bother, will fix this later.